server.js

```diff
@@ -1,35 +1,110 @@
-import express from "express";
-import cors from "cors";
-import mongoose from "mongoose";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+import express from 'express';
+import cors from 'cors';
+import mongoose from 'mongoose';
+import goldenGlobesData from './data/golden-globes.json';
+
+const data = goldenGlobesData;
+
+const mongoUrl = process.env.MONGO_URL || 'mongodb://localhost/project-mongo';
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
-const port = process.env.PORT || 8080;
+const port = process.env.PORT || 8090;
 const app = express();
 
+// Starts populating the database
+const Award = mongoose.model('Award', {
+	year_film: Number,
+	year_award: Number,
+	ceremony: Number,
+	category: String,
+	nominee: String,
+	film: String,
+	win: Boolean,
+});
+
+//Starts seeding the database
+if (process.env.RESET_DB) {
+	const seedDatabase = async () => {
+		await Award.deleteMany({});
+		data.forEach((item) => {
+			const newAward = new Award(item);
+			newAward.save();
+		});
+	};
+	seedDatabase();
+}
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
+// Middleware that checks if the database is connected before going to endpoints
+app.use((req, res, next) => {
+	if (mongoose.connection.readyState === 1) {
+		next();
+	} else {
+		res.status(503).json({ error: 'Service unavailable' });
+	}
+});
+
+//RESTful routes/endpoints
+app.get('/', (req, res) => {
+	res.send(
+		'Welcome to the Golden-Globes API. Endpoints: /goldenglobes /nominations'
+	);
+});
+
+//this route will display all nominations
+app.get('/nominations', (req, res) => {
+	res.json(data);
```

Comment:
If you want you can add some error handling here, like you have below. Maybe also add an else if for both success and for error (staus(200) and status(400) for example.